pkg/server/grpc/tracee.go

```diff
@@ -42,10 +42,14 @@ func (s *TraceeService) StreamEvents(in *pb.StreamEventsRequest, grpcStream pb.T
 	for e := range stream.ReceiveEvents() {
 		// TODO: this conversion is temporary, we will use the new event structure
 		// on tracee internals, so the event received by the stream will already be a proto
-		eventProto := convertTraceeEventToProto(e)
+		eventProto, err := convertTraceeEventToProto(e)
+		if err != nil {
+			return err
+		}
+
 		mask.Filter(eventProto)
 
-		err := grpcStream.Send(&pb.StreamEventsResponse{Event: eventProto})
+		err = grpcStream.Send(&pb.StreamEventsResponse{Event: eventProto})
 		if err != nil {
 			return err
 		}
```

Comment:
ditto

Reply:
I don't think so, if we can't send is because the connection on the other side isn't there anymore for a a given reason, so we should stop sending and unsubscribe the client. WDYT?

Reply:
yes, sounds reasonable